service/routes/rum.js

```diff
@@ -0,0 +1,125 @@
+/* Endpoints for Real User Monitoring of detect and perf results */
+
+'use strict';
+
+const polyfillio = require('../../lib');
+const metrics = require('../metrics');
+const express = require('express');
+
+let mysql;
+if (process.env.MYSQL_DSN) {
+	require('promise-mysql').createConnection(process.env.MYSQL_DSN).then(conn => {
+		mysql = conn;
+	});
+}
```

Comment:
If the connection fails to be made, should we catch that failure and either crash the process with an informative error or write to stdout/logs that it failed and carry-on?

I think a combination of the two would work best. Write to stdout and carry-on if not in production, crash process with informative error if in production.